demo/row-filtering-example.html

```diff
@@ -762,13 +762,18 @@
                 "submit_buttons": on_submit,
                 "subtasks": subtasks,
                 "toolbox_order": [0, 1, 3, 6, 8, 2, 7, 4, 5],
-                "config_data": {
-                    "distance_filter_toolbox_item": {
-                        "show_overlay": true,
-                        // "disable_multi_class_mode": true,
-                        // "filter_during_polyline_move": false,
-                    }
-                }
+                "distance_filter_toolbox_item": {
+                    "default_values": {
+                        "closest_row": {
+                            "distance": 200, // px
+                        }
+                    },
+                    "show_overlay": true,
+                    // "disable_multi_class_mode": true,
+                    // "filter_during_polyline_move": false,
+                },
+                "keypoint_slider_default_value": 0.2,
+                "filter_annotations_on_load": true,
```

Comment:
honestly, im not convinced you would provide a default value but not want to filter on load. Thoughts?

Reply:
It definitely could happen - consider a case where the sole task of an annotator is to decide whether or not to apply a filter, and they're instructed to lean towards not applying it. You'd want to provide a sensible default for them to toggle, but start without filtering (and probably also have a good keybind for toggling the filtering).

I would concede that most physical examples I could contrive that fit this could be trivially automated. I'd still keep this. Worst case is that it's a default that is never touched.

Reply:
yeah we should probably keep it for completeness. Imma switch the default to `true` tho since when i was trying to use it i had assumed that provided the default would trigger on load.

Reply:
[a73f1d8](https://github.com/SenteraLLC/ulabel/pull/170/commits/a73f1d8405fed74f6902eda2d080a75eca6dfd5d)

Reply:
it will only trigger if the Keypoint Slider is present, since it occurs in its constructor, which is only called if it is listed in the `toolbox_order`